### PR TITLE
chore(sdk): move console tests to functional

### PR DESCRIPTION
**Pull request by kptkin**
Description
-----------

What does the PR do? Include a concise description of the PR contents.

- [x] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

File: tests/system_tests/test_functional/console/test_console.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        run = wandb_init(settings={"console": "auto"})
+        progress_bar = tqdm.tqdm(range(5))
+        progress_bar.update(2)
+        run.finish()
+        progress_bar.update(1)
+
+    output = relay.context.output[0]
+    assert output[0].startswith("ERROR") and bool(
+        re.search(r"40%\|█+\| 2/5", output[0])
+    )
```
Comment (timoffex) [inline tests/system_tests/test_functional/console/test_console.py:82]: instead of skipping, why not delete or fix?

Comment (kptkin) [reply tests/system_tests/test_functional/console/test_console.py:82]: i think we should fix it, but this PR just moves test out of the yea folder. There should be a different task to actually fix these tests

**Review by dmitryduev — APPROVED** [review]

File: tests/system_tests/test_functional/console/test_console.py (hunk 1 above)
Comment (timoffex) [reply tests/system_tests/test_functional/console/test_console.py:82]: that's fine, but isn't keeping the test in the yea folder a good way to track that work?

Comment (kptkin) [reply tests/system_tests/test_functional/console/test_console.py:82]: i'd rather just have it tracked in Jira and delete that folder, as soon as possible

**Pull request merged**